proto/buf/alpha/registry/v1alpha1/repository_track_commit.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// RepositoryTrackCommit is the existance of a RepositoryCommit on a RepositoryTrack. Currently its only purpose is
+// for querying whether a RepositoryCommit is on a RepositoryTrack and determining it's sequence id.
+message RepositoryTrackCommit {
+  // We reserve field number '3' for id, but RepositoryTrackCommits have no id.
+  // string id = 1;
+
+  // immutable
+  google.protobuf.Timestamp create_time = 2;
+
+  // We reserve field number '3' for the update_time.
+  // google.protobuf.Timestamp update_time = 3;
+
+  // unique for repository_track, immutable
+  int64 sequence_id = 4;
+}
```

Comment:
I suppose we don't need it to start, but I imagine we might need this object to include the ID for the commit it's associated with. You could say the same for the track's ID.

In short, we effectively want to expose the public-equivalent of our internal representation. 

```
message RepositoryTrackCommit {
  // immutable
  bytes repository_track_entity_id = 1;
  // immutable
  google.protobuf.Timestamp create_time = 2;
  // mutable
  google.protobuf.Timestamp update_time = 3;
  // unique for repository_track_entity_id, immutable
  bytes repository_commit_entity_id = 4;
  // unique for repository_track_entity_id, immutable
  int64 sequence_id = 5;
}
```

So in this case, the entity IDs, would be the public IDs.